notebooks/notebooks/iclr2022/estimate-flops-katago.ipynb

```diff
@@ -0,0 +1,662 @@
+{
```

Comment:
<p>It's impressive the results from thop and ptflops are so similar (and low SD amongst themselves). Thanks for being rigorous and cross-checking them!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/HumanCompatibleAI/KataGoVisualizer/pull/40/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
Haha thanks. https://www.lesswrong.com/posts/jJApGWG95495pYM7C/how-to-measure-flop-s-for-neural-networks-empirically was very helpful for determining which libraries to use.